internal/security/credentials_test.go

```diff
@@ -40,86 +42,123 @@ type getSecretsTestObj struct {
 	resetSecretsCache bool
 }
 
-var getSecretsTestData = []getSecretsTestObj{
-
-	getSecretsTestObj{
-		testName:          "Empty keys",
-		path:              "db_secrets",
-		keys:              []string{"", ""},
-		expectedSecrets:   nil,
-		expectedErr:       errors.New("Couldn't get secrets: No value for the keys: [] exists"),
-		resetSecretsCache: true,
-	},
-
-	getSecretsTestObj{
-		testName:          "One valid key, one empty key",
-		path:              "db_secrets",
-		keys:              []string{"key1", ""},
-		expectedSecrets:   nil,
-		expectedErr:       errors.New("Couldn't get secrets: No value for the keys: [] exists"),
-		resetSecretsCache: true,
-	},
-	getSecretsTestObj{
-		testName:          "One valid key one not found key",
-		path:              "db_secrets",
-		keys:              []string{"key1", "notFoundKey"},
-		expectedSecrets:   nil,
-		expectedErr:       errors.New("Couldn't get secrets: No value for the keys: [notFoundKey] exists"),
-		resetSecretsCache: true,
-	},
-	getSecretsTestObj{
-		testName:          "Not found key",
-		path:              "db_secrets",
-		keys:              []string{"notFoundKey"},
-		expectedSecrets:   nil,
-		expectedErr:       errors.New("Couldn't get secrets: No value for the keys: [notFoundKey] exists"),
-		resetSecretsCache: true,
-	},
-	getSecretsTestObj{
-		testName:          "Valid key",
-		path:              "db_secrets",
-		keys:              []string{"key1"},
-		expectedSecrets:   map[string]string{"key1": "value1"},
-		resetSecretsCache: true,
-		expectedErr:       nil,
-	},
-	getSecretsTestObj{
-		testName:          "Two valid keys",
-		path:              "db_secrets",
-		keys:              []string{"key1", "key2"},
-		expectedSecrets:   map[string]string{"key1": "value1", "key2": "value2"},
-		resetSecretsCache: true,
-		expectedErr:       nil,
-	},
-	getSecretsTestObj{
-		testName:          "Valid key (key1 not cached)",
-		path:              "db_secrets",
-		keys:              []string{"key1"},
-		expectedSecrets:   map[string]string{"key1": "value1"},
-		expectedErr:       nil,
-		resetSecretsCache: false,
-	},
-	getSecretsTestObj{
-		testName:          "One valid key (key1 cached)",
-		path:              "db_secrets",
-		keys:              []string{"key1"},
-		expectedSecrets:   map[string]string{"key1": "value1"},
-		expectedErr:       nil,
-		resetSecretsCache: false,
-	},
-	getSecretsTestObj{
-		testName:          "Two valid keys (key1 cached, key2 not cached)",
-		path:              "db_secrets",
-		keys:              []string{"key1", "key2"},
-		expectedSecrets:   map[string]string{"key1": "value1", "key2": "value2"},
-		expectedErr:       nil,
-		resetSecretsCache: false,
-	},
+var getSecretsTestData []getSecretsTestObj
```

Comment:
Remove the unused init() func at the beginning: https://github.com/edgexfoundry/app-functions-sdk-go/pull/285/files#diff-660b183c6291f1c689e70b7832f651adR29-R30